Documentation/gettingstarted/k8s-install-helm.rst

```diff
@@ -174,10 +174,23 @@ Install Cilium
           Kubernetes worker node than the ENI limit, but means that pod
           connectivity to resources outside the cluster (e.g., VMs in the VPC
           or AWS managed services) is masqueraded (i.e., SNAT) by Cilium to use
-          the VPC IP address of the Kubernetes worker node.  Excluding the
-          lines for ``eni.enabled=true``, ``ipam.mode=eni`` and
-          ``tunnel=disabled`` from the helm command will configure Cilium to
-          use overlay routing mode (which is the helm default).
+          the VPC IP address of the Kubernetes worker node. To set up Cilium 
+          overlay mode, follow the steps below:
+
+            1. Excluding the lines for ``eni.enabled=true``, ``ipam.mode=eni`` and 
+               ``tunnel=disabled`` from the helm command will configure Cilium to use 
+               overlay routing mode (which is the helm default).
+            2. Flush iptables to clear any iptables configurations done by VPC CNI
+
+               .. code-block:: shell-session
+               
+                  iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
```

Comment:
I think it would be safer if we detected the VPC CNI rules and only deleted those. This command will delete much more than just VPC CNI AFAIK.